features/config.feature

```diff
@@ -92,6 +92,144 @@ Feature: Config
       The requested config file does not exist
       """
 
+  Scenario: PHP configuration file
+    Given a file named "behat.php" with:
+      """
+      <?php
+
+      use Behat\Config\Config;
+
+      return new Config();
```

Comment:
I would ideally want to see each of these config examples actually have some config so that it is clear it has been parsed and applied rather than just the file found. 

I realise the existing yaml scenarios in this file don't do that, on the other hand all the other features in our suite use yaml config files so we have a lot of coverage of them being applied elsewhere.

The easiest / least verbose proof would probably be to set it to use a non-default formatter e.g. `progress`.

Reply:
done